influx/__init__.py

```diff
@@ -27,10 +30,10 @@ def _get_client():
     global _telegraf_client
 
     if not _statsd_influx_host:
-        raise MissingConfiguration('Missing STATSD_INFLUX_HOST setting')
```

Comment:
@siluka is this fine if it's just a warning? Or did you have any special reason for an exception here?